README.md

```diff
@@ -20,24 +20,28 @@ This is a patch for happybase to support kerberos when connect to hbase thrift s
 
 The patch now support `python 2.7`, `python 3.5`, `python 3.6`, `python 3.7`.
 
+The project is base on happybase https://github.com/python-happybase/happybase
+
+
 # requirements
 
 In order to use kerberos, the following packages is required:
 
-1. python3-dev
-2. libkrb5-dev
+1. python-devel
 
-For example, in ubuntu install by the follow commands:
+For example, in redhat install by the follow commands:
```

Comment:
Maybe you could add another example for redhat alone and keep the ubuntu example unchanged.